### PR TITLE
Stupid voxslugs dies to 2 hits by your feets

### PR DESCRIPTION
**Pull request by Rayfod**
They are dumb weak right now, so them getting a buff will do them good

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm [hunk 1]
```diff
@@ -13,17 +13,17 @@ Small, little HP, poisonous.
...
 	speed = 0
 	move_to_delay = 0
 	density = 1
 	min_gas = null
 	mob_size = MOB_MINISCULE
 	can_escape = TRUE
 	pass_flags = PASS_FLAG_TABLE
-	melee_damage_lower = 5
-	melee_damage_upper = 10
+	melee_damage_lower = 15
```
Comment (Kaksisilma) [inline code/modules/mob/living/simple_animal/hostile/voxslug.dm]:
```suggestion
	melee_damage_lower = 10
```

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm [hunk 2]
```diff
@@ -13,17 +13,17 @@ Small, little HP, poisonous.
...
+	melee_damage_upper = 30
```
Comment (Kaksisilma) [inline code/modules/mob/living/simple_animal/hostile/voxslug.dm]:
```suggestion
	melee_damage_upper = 20
```

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm [hunk 3]
```diff
@@ -13,17 +13,17 @@ Small, little HP, poisonous.
 	response_disarm = "gently pushes aside"
 	response_harm   = "stamps on"
 	destroy_surroundings = 0
-	health = 15
-	maxHealth = 15
+	health = 60
+	maxHealth = 60
```
Comment (Kaksisilma) [inline code/modules/mob/living/simple_animal/hostile/voxslug.dm]:
```suggestion
	maxHealth = 40
```

File: code/modules/mob/living/simple_animal/hostile/voxslug.dm [hunk 4]
```diff
@@ -13,17 +13,17 @@ Small, little HP, poisonous.
 	response_disarm = "gently pushes aside"
 	response_harm   = "stamps on"
 	destroy_surroundings = 0
-	health = 15
-	maxHealth = 15
+	health = 60
```
Comment (Kaksisilma) [inline code/modules/mob/living/simple_animal/hostile/voxslug.dm]:
```suggestion
	health = 40
```

**Review by Kaksisilma — CHANGES_REQUESTED** [review]
Not against giving them a buff, but this is really overtuned as is.

**Comment by Rayfod** [comment]
Changes added

**Review by Kaksisilma — APPROVED** [review]

**Pull request merged**